client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -206,6 +226,16 @@ class CheckoutThankYouHeader extends PureComponent {
 			);
 		}
 
+		if ( isDelayedDomainTransfer( primaryPurchase ) ) {
+			return (
+				<div className="checkout-thank-you__header-button">
+					<button className={ headerButtonClassName } onClick={ this.startTransfer }>
+						{ translate( 'Start the transfer' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Start Transfer Again' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/thank-you-delayed-transfer)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).